src/06-sort.js

```diff
@@ -33,7 +40,14 @@ function sortByArtistNameDescending(songs) {}
  * @param {Object[]} songs - An array of songs. See the song data for more.
  * @returns {Object[]}
  */
-function sortBySongTitleAscending(songs) {}
+const sortBySongTitleAscending = (songs) =>
+  songs.sort((a, b) =>
+    b.title.toLowerCase() === a.title.toLowerCase()
```

Comment:
This is clever, but it is not best practice. Ternary is suppose to be simple. More often then not it will be a simple one liner.